README.md

```diff
@@ -159,7 +159,7 @@ function expired(bytes32 caseId,uint8 status) returns (bool);
 
 /// @return bool - true if it actually start counting for the grace period
 
-///                false -if the grace period already started
+///                false if the grace period already started
 
 function startGracePeriod(bytes32 caseId,uint8 status) returns (bool);
 
```

Comment:
All of the above has simply been copy+pasted from the abstract contracts, ignoring that its not formatted or anything here. That is not what a README is for. You havent even gone into more detail or explained the reasons for the implementation choices.